scripts/scratch/16-initialize-non-aragon-contracts.js

```diff
@@ -125,14 +125,26 @@ async function deployNewContracts({ web3, artifacts }) {
   await makeTx(validatorsExitBusOracle, 'initialize', validatorsExitBusOracleArgs, { from: DEPLOYER })
 
   //
-  // === WithdrawalQueue initialize ===
+  // === WithdrawalQueue: initialize ===
   //
+  const withdrawalQueue = await artifacts.require('WithdrawalQueueERC721').at(withdrawalQueueAddress)
   const withdrawalQueueArgs = [
     withdrawalQueueAdmin,  // _admin
   ]
-  const withdrawalQueue = await artifacts.require('WithdrawalQueueERC721').at(withdrawalQueueAddress)
   await makeTx(withdrawalQueue, 'initialize', withdrawalQueueArgs, { from: DEPLOYER })
 
+  //
+  // === WithdrawalQueue: setBaseURI ===
+  //
+  const withdrawalQueueBaseUri = state["withdrawalQueueERC721"].deployParameters.baseUri
+  if (withdrawalQueueBaseUri !== null && withdrawalQueueBaseUri !== "") {
+    const MANAGE_TOKEN_URI_ROLE = await withdrawalQueue.MANAGE_TOKEN_URI_ROLE()
+    await makeTx(withdrawalQueue, 'grantRole', [MANAGE_TOKEN_URI_ROLE, DEPLOYER], { from: DEPLOYER })
```

Comment:
Is it easy to make it one tx?

Reply:
Would like to see it atomic indeed, but don't know a simple nice solution, thb

In general, think having UpgradeTemplate-like helper contract might be the way

Reply:
It's possible by utilizing the Multicall contract that is usually deployed everywhere and well-known. Ape/brownie even have a built-in support for that.

Reply:
I'm not sure it works when the call goes from EOA-deployer which is the admin due to 

> Because you cannot delegatecall from an EOA, this significantly reduces the benefit of calling Multicall3 from an EOA—any calls the Multicall3 executes will have the MultiCall3 address as the msg.sender. This means you should only call Multicall3 from an EOA if the msg.sender does not matter.

from https://github.com/mds1/multicall#batch-contract-writes

Is it?

Reply:
Yep. Correct. Let it be separate txs for now. 

But, the audited deployer/template contract may be worth considering in the next iteration.

Reply:
I think even unaudited one is a worthy improvement.